fairscale/experimental/nn/ssd_offload.py

```diff
@@ -90,22 +94,26 @@ class SsdTensorHandle(torch.Tensor):
     point in time the Tensor may be in memory or on disk.
 
     Args:
-        shape Tuple[int, ...]: Shape of the tensor that is represented by the handle.
+        shape torch.Size: Shape of the tensor that is represented by the handle.
         dtype: torch.dtype: Dtype of the tensor that is represented by the handle.
         requires_grad: bool: Property of the tensor that is represeneted by the handle.
 
     Returns:
         A SSDTensorHandle object representing a Tensor.
     """
 
+    override_directory_path: Optional[str] = None
+
     @staticmethod
     def __new__(
-        cls: SsdTensorHandle, shape: Tuple[int, ...], dtype: torch.dtype, requires_grad: bool = False
-    ) -> SsdTensorHandle:
-        r = torch.Tensor._make_wrapper_subclass(cls, shape, dtype=dtype, requires_grad=requires_grad)  # type: ignore
+        cls: Type[SsdTensorHandle], shape: torch.Size, dtype: torch.dtype, requires_grad: bool = False
+    ) -> "SsdTensorHandle":
```

Comment:
why do we need quotes here for specifying the return type?

Reply:
I remember that mypy had some issues with the function signature of this class before. Some things have changed (including mypy version that I developed this code on, and it doesn't seem to be having problems with the return type not being inclosed in quotes. I will remove them in the next version of this PR.